system-ballerina/Ballerina.toml

```diff
@@ -14,8 +17,8 @@ target = "java8"
 
     [[platform.libraries]]
     artifactId = "ballerina-io"
-    version = "@lang.version@"
-    path = "./lib/ballerina-io-@lang.version@.jar"
+    version = "2.0.0-Preview4-SNAPSHOT"
```

Comment:
This change should be temporary during the build

Reply:
Added it as a gradle property.